scripts/tests.sh

```diff
@@ -141,6 +155,9 @@ then
     exit 1
 fi
 
-pkill "$ETH_PID" || true
-sleep 4
-pgrep "$ETH_PID" && pkill -9 "$ETH_PID" || true
+if [ "$IPC_ENABLED" = true ]
```

Comment:
Is this correct with a single equation sign?

Reply:
Yes, single equality performs string comparison. It might be better to use `"true"`, though to reduce the confusion.

Reply:
So this above `if [[ "$OSTYPE" == "darwin"* ]]` should be simplified to `+if [ "$OSTYPE" = "darwin"* ]` ?

Reply:
``[[ "$OSTYPE" = "darwin"* ]]`` will work, but single brackets won't.  ``[`` is the good old shell ``test``, ``[[`` also supports regex, which are in use here: ``"darwin"*``.